services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowSyncSpec.groovy

```diff
@@ -33,17 +33,13 @@ class FlowSyncSpec extends HealthCheckSpecification {
         def flowPath = PathHelper.convert(northbound.getFlowPath(flow.id))
 
         def involvedSwitches = pathHelper.getInvolvedSwitches(flow.id)
-        def ruleToDelete = getFlowRules(involvedSwitches.first()).first().cookie
-        involvedSwitches.each { northbound.deleteSwitchRules(it.dpId, ruleToDelete) }
-        involvedSwitches.each { sw ->
-            Wrappers.wait(RULES_DELETION_TIME) { assert getFlowRules(sw).size() == flowRulesCount - 1 }
+        List<Long> rulesToDelete = getFlowRules(switchPair.src)*.cookie
```

Comment:
you forgot to update the `given` step description(` "An intermediate-switch flow with one deleted rule on each switch"`)
actually you delete  rules on the src switch only

Reply:
fxd